src/components/popup.jsx

```diff
@@ -46,6 +54,13 @@ export default class Popup extends React.Component {
     });
   }
 
+  showCustomInput() {
+    this.setState({
+      activeColor: '',
```

Comment:
So I was trying to figure out why the state wouldn't persist the color and finally found this. In order to maintain the hex value in the input field, we need to remove this line. I think this is important for users, so that they can maintain what hex value they've currently entered.